src/nvim/normal.c

```diff
@@ -2105,10 +2105,8 @@ bool do_mouse(oparg_T *oap, int c, int dir, long count, bool fixindent)
     } else {  // MOUSE_RIGHT
       stuffcharReadbuff('#');
     }
-  }
-  // Handle double clicks, unless on status line
-  else if (in_status_line) {
-  } else if (in_sep_line) {
+  } else if (in_status_line) {  // Handle double clicks, unless on status line
+  } else if (in_sep_line) {     // Mouse in vertical separator line - skip
```

Comment:
```suggestion
  } else if (in_status_line || in_sep_line) {
    // Do nothing if on status line or vertical separator
    // Handle double clicks otherwise
```